Makefile

```diff
@@ -45,8 +45,8 @@ format: $(FMT)
 vet: $(GO)
 	$(GO) vet ./pkg/...
 
-test: $(GO) vet format
-	$(GO) test -timeout 2m -v ./pkg/...
+test: $(GO)
```

Comment:
if you remove  `vet` and `format` from here then they are not used at all. maybe we should move them to another dependency so it would checked regularly?

Reply:
I am going to put them back and fix this at a different PR, so we do similar that we do with KMP running them at CI and checking that they don't change code.